index.html

```diff
@@ -5,8 +5,59 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
+    <header>
+      <nav>
+        <a>
+          <img src="img.img" width="138" height="70" alt="Логотип Sedona">
+        </a>
+        <ul>
+          <li><a href="">Информация</a></li>
+          <li><a href="">Фото и видео</a></li>
+          <li><a href="">Карта штата</a></li>
+          <li><a href="hotels.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div>
+        <img src="img/sedona.svg" alt="Добро пожаловать в Седону">
+      </div>
+      <h1> Sedona </h1>
+      <section>
+        <h2>Седона - небольшой городок в Аризоне,<br>
+        заслуживающий большего!
+        </h2>
+        <p>Рассмотрим 5 причин, по которым Седона лучше, чем Гранд Каньон!</p>
+      </section>
+      <section>
+        <h3>Заинтересовались?</h3>
+        <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <p>
+          <button type='button'>Поиск гостиницы в Седоне</button>
+          <!-- Здесь будет форма
+          Дата заезда: <time></time><span></span><br>
+          Дата выезда: <time></time><span></span><br>
+          Взрослые: <span> </span> Дети: <span></span><br>-->
+          <button type='button'>Найти</button>
```

Comment:
```suggestion
          <button type="submit">Найти</button>
```
Пока лучше убрать, так как данная кнопка является элементом формы.